install/kubernetes/helm/istio/charts/security/templates/enable-mesh-mtls.yaml

```diff
@@ -4,18 +4,21 @@
 # a separated yaml file so that customer can enable mTLS independent from installation.
 
 # Authentication policy to enable mutual TLS for all services (that have sidecar) in the mesh.
-apiVersion: "authentication.istio.io/v1alpha1"
-kind: "MeshPolicy"
+apiVersion: "security.istio.io/v1beta1"
+kind: "PeerAuthentication"
 metadata:
   name: "default"
+  namespace: {{ .Release.Namespace }}
+spec:
```

Comment:
this looks wrong. 

Also, again, I don't think we should modify this folder. It is deprecated/frozen and no new changes are accepted

Reply:
Removed.